bindings/python/src/lister.rs

```diff
@@ -42,7 +42,12 @@ impl BlockingLister {
     }
     fn __next__(mut slf: PyRefMut<'_, Self>) -> PyResult<Option<PyObject>> {
         match slf.0.next() {
-            Some(Ok(entry)) => Ok(Some(Entry::new(entry).into_py(slf.py()))),
+            Some(Ok(entry)) => Ok(Some(
+                Entry::new(entry)
+                    .into_pyobject(slf.py())?
+                    .into_any()
+                    .unbind(),
```

Comment:
I think these can be simplified a bit by using https://docs.rs/pyo3/latest/pyo3/conversion/trait.IntoPyObjectExt.html#method.into_py_any